### ISSUE TITLE
bot: Create move command

### ISSUE THREAD

**Issue by rsodre** [underware-gg/pistols-discord-bot#9]
Depends on #8

## Tasks

- [ ] Create new command: `move`
  - [ ] Input: `duel_id`
  - [ ] Input: `action`

- [ ] Allow players to make a move on duel listing commands
  - [ ] Identify all commands that list duels
  - [ ] Identify if `duelist_a` and `duelist_b` are linked to Discord
  - [ ] If said duelists are in turn to make a move, display an additional private message to them
  - [ ] The message must contain a link that triggers the `move` command, with `duel_id` already set

## Notes

* We're still planning out how this will work
* This issue is **ON HOLD**
